Source/BugsnagClient.m

```diff
@@ -811,14 +815,26 @@ - (void)orientationChanged:(NSNotification *)notif {
         [orientationNotifName isEqualToString:lastBreadcrumb[BSGKeyName]]) {
         NSDictionary *metadata = lastBreadcrumb[BSGKeyMetadata];
 
-        if ([orientation isEqualToString:metadata[BSGKeyOrientation]]) {
+        if ([orientation isEqualToString:metadata[BSGKeyOrientation]])
             return; // ignore duplicate orientation event
-        }
     }
 
-    [[self state] addAttribute:BSGKeyOrientation
-                     withValue:orientation
-                 toTabWithName:BSGKeyDeviceState];
+    // It's not a change
+    if ([orientation isEqualToString:lastOrientation])
```

Comment:
nit: could braces be added to new if statements throughout this file?

I don't know if we actually have any official code style guide for Cocoa, so maybe that's something to discuss.

Reply:
We don't.  I've been taking the arbitrary decision that single line conditionals don\t need the baggage of a closing brace.  I can see the other side of the argument.  I'll leave as is and raise a ticket to come up with a definitive style guide.

Reply:
The style guide for C code is pretty universally always include braces around if statements because it's easy to miss that they are missing when adding future code.

Reply:
Task created to agree on Cocoa style.  Braces will be readded.